src/status_im/bots/models.cljs

```diff
@@ -0,0 +1,15 @@
+(ns status-im.bots.models)
```

Comment:
Well, we need to divide event handling logic into multiple namespaces, it's quite complex and namespaces like `chat.events` and `chat.events.input` are already 500+ LOC, so I would argue against merging them. 
One way how to solve the circular dependency thing without separating pure functions into model namespaces would be to require all distinct events namespaces in some top level (event) ns which is loaded upon startup, like `status-im.ui.screens.events`.
Disadvantage against current way where "parent" event namespaces require "child" event namespaces just to get events registered, without any actual dependency on "child" ns code (bad code smell, I agree) would be that `:require` vector in that very top level event namespace would be huge.